extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -279,6 +271,19 @@ export class SchemaCompareResult {
 			this.flexModel.addItem(this.noDifferencesLabel, { CSSStyles: { 'margin': 'auto' } });
 		}
 
+		// explicitly exclude things that were excluded in previous compare
+		const thingsToExclude = this.sourceTargetSwitched ? this.originalTargetExcludes : this.originalSourceExcludes;
+		if (thingsToExclude) {
+			thingsToExclude.forEach(item => {
+				service.schemaCompareIncludeExcludeNode(this.comparisonResult.operationId, item, false, azdata.TaskExecutionMode.execute);
+			});
```

Comment:
there is one more call for this schemaCompareIncludeExcludeNode() when boxes are unchecked we need toggle there too

Reply:
Yes there is a check so the button states are changed to disabled below this everything is excluded. Or did you mean another place that this needs to be checked?

Reply:
Oh got it... I was thinking that that place where actual check box gets checked/unchecked - we are missing the toggle call there but realized that saveState is called from there so having the toggle inside saveState makes sense